bitnami/kafka/templates/scripts-configmap.yaml

```
@@ -93,7 +93,7 @@ data:
         fi
     fi
 
-    if [[ $KAFKA_CFG_PROCESS_ROLES == *"controller"* ]]; then
+    if [[ $KAFKA_CFG_PROCESS_ROLES == *"controller"* ]] && [[ -z "${KAFKA_CFG_CONTROLLER_QUORUM_VOTERS}" ]]; then
```

Since this is running in bash, it could be simplified a bit:

```suggestion
    if [[ $KAFKA_CFG_PROCESS_ROLES == *"controller"* && -z $KAFKA_CFG_CONTROLLER_QUORUM_VOTERS ]]; then
```

---

Sure thing, I will make the change now. Thanks for the feedback.